### ISSUE TITLE
Cannot export EfficientNet to onnx

### ISSUE THREAD

**Issue by LamyaMohaned** [lukemelas/EfficientNet-PyTorch#296]
Hello

I used the same script for exporting a model to onnx, but I got this error:

> RuntimeError: ONNX export failed: Couldn't export Python operator SwishImplementation

when I added this part:
`module.set_swish(memory_efficient=False)`, I got this error:

> RuntimeError: Exporting the operator silu to ONNX opset version 9 is not supported. Please open a bug to request ONNX export support for the missing operator.

**Comment by LamyaMohaned** [comment]
The problem is solved by upgrading pytorch to 1.8.1

**Comment by ShunLu91** [comment]
Hi, 
I solved the above issue and managed to export the onnx model. 
However, I failed to load the onnx model when transform it to the tensorrt format. Have you ever encountered  such issue?
```shell
In node -1 (importPad): UNSUPPORTED_NODE: Assertion failed: inputs.at(1).is_weights()
```
